endpoints/globus/endpoint_test.go

```diff
@@ -274,8 +274,10 @@ func TestGlobusTransferCancellation(t *testing.T) {
 // this runs setup, runs all tests, and does breakdown
 func TestMain(m *testing.M) {
 	var status int
-	setup()
-	status = m.Run()
-	breakdown()
+	//setup()
+	//status = m.Run()
+	//breakdown()
+	fmt.Printf("Skipping Globus endpoints tests because LBNL's Google Drive connector is down!")
+	status = 0
```

Comment:
fingers crossed this is a temporary problem, but if it looks like this might be something that happens regularly, we may want to have something more robust in place to enable or disable various tests.

Reply:
Yeah, I'm with you crossing my fingers. If it does turn out to be A Thing, we can discuss how to flip things on and off.